images/ubuntu/scripts/build/Configure-Toolset.ps1

```diff
@@ -4,91 +4,79 @@
 ##  Desc:  Configure toolset
 ################################################################################
 
-Import-Module "$env:HELPER_SCRIPTS/../tests/Helpers.psm1" -DisableNameChecking
-
-function Get-ToolsetToolFullPath
-{
-    param
-    (
-        [Parameter(Mandatory)] [string] $ToolName,
-        [Parameter(Mandatory)] [string] $ToolVersion,
-        [Parameter(Mandatory)] [string] $ToolArchitecture
+Import-Module "$env:HELPER_SCRIPTS/../tests/Helpers.psm1"
+
+function Get-TCToolVersionPath {
+    param(
+        [Parameter(Mandatory)]
+        [string] $ToolName,
+        [Parameter(Mandatory)]
+        [string] $ToolVersion,
+        [Parameter(Mandatory)]
+        [string] $ToolArchitecture
     )
 
-    $toolPath = Join-Path -Path $env:AGENT_TOOLSDIRECTORY -ChildPath $toolName
-    $toolPathVersion = Join-Path -Path $toolPath -ChildPath $toolVersion
-    $foundVersion = Get-Item $toolPathVersion | Sort-Object -Property {[version]$_.name} -Descending | Select-Object -First 1
-    $installationDir = Join-Path -Path $foundVersion -ChildPath $toolArchitecture
+    $toolPath = Join-Path -Path $env:AGENT_TOOLSDIRECTORY -ChildPath $ToolName
+    $toolPathVersion = Join-Path -Path $toolPath -ChildPath $ToolVersion
+    $foundVersion = Get-Item $toolPathVersion | Sort-Object -Property { [version] $_.name } -Descending | Select-Object -First 1
+    $installationDir = Join-Path -Path $foundVersion -ChildPath $ToolArchitecture
+
     return $installationDir
 }
 
-function Add-EnvironmentVariable
-{
-    param
-    (
-        [Parameter(Mandatory)] [string] $Name,
-        [Parameter(Mandatory)] [string] $Value,
+function Add-ETCEnvironmentVariable {
```

Comment:
I would not pin the function name to etcSomething, etc is just a path, I'd call it `Add-GlobalEnvironmentVariable`